### PR TITLE
Iavp merge 1444 1416

### PR DESCRIPTION
**Pull request by Jrad7**

### CODE REVIEW

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 1]
```diff
@@ -76,7 +76,7 @@ sub generate_advice {
 
         # These checks will only run on v88 and highger.
         if ( Cpanel::Version::compare( $cpanel_version, '>=', $IMUNIFYAV_MINIMUM_CPWHM_VERSION )
-            && ( !$self->{i360}{installed} && !$self->{i360}{licensed} ) ) {
+            && !$self->{i360}{installed} ) {
```
Comment (atoomic) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm:79]: we should probably check for `$self->{i360}` also

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm [hunk 2]
```diff
@@ -290,10 +260,26 @@ sub _suggest_imunify360 {
 sub _suggest_iav {
     my ($self) = @_;
 
-    if ( !$self->{iav}{installed} && !$self->{iavp}{licensed} ) {
+    if ( $self->{iav}{installed} && !$self->{iavp}{licensed} ) {
```
Comment (atoomic) [inline pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm:263]: maybe easier to read by using a block `if ( !$self->{iavp}{licensed} ){ ....` but just one idea
the two if / else if could then be inside that block 
nothing wrong there

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 1 above)
Comment (Jrad7) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm:80]: done

File: pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm (hunk 2 above)
Comment (Jrad7) [reply pkg/Cpanel/Security/Advisor/Assessors/Imunify360.pm]: done

**Pull request merged**